curriculum/challenges/english/01-responsive-web-design/css-grid-magazine/part-004.md

```diff
@@ -0,0 +1,154 @@
+---
+id: 6143869bb45bd85e3b1926aa
+title: Part 4
+challengeType: 0
+dashedName: part-4
+---
+
+# --description--
+
+Within your `.heading` element, create a `header` element with the `class` set to `hero`.
+
+In that element, create an `img` element with the `src` set to `https://cdn.freecodecamp.org/platform/universal/fcc_meta_1920X1080-indigo.png`, the `alt` set to `freecodecamp logo`, the `loading` attribute set to `lazy`, and the `class` set to `hero-img`.
+
+After your `img` element, add an `h1` element with the `class` set to `hero-title` and the text set to `OUR NEW CURRICULUM`, followed by a `p` element with the `class` set to `hero-subtitle` and the text set to `Our efforts to restructure our curriculum with a more project-based focus`.
```

Comment:
Can we divide this into 2 different sentces

Reply:
To clarify, break the text up a bit more?

Or are you meaning two separate lessons?